images/kubevirt-ci-testenv/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/k8s-testimages/bootstrap
+
+RUN mkdir -p /opt/source/
+#RUN yum install -y python3-virtualenv python3-libselinux git make
```

Comment:
I guess you can remove this?

Reply:
I guess I can, if everything will be tested with debian upstream.

Reply:
No, I menat you can remove this commented out line.